src/envoy/http/authn/authn_utils_test.cc

```diff
@@ -116,11 +117,13 @@ TEST(AuthnUtilsTest, GetJwtPayloadFromHeaderTest) {
   EXPECT_TRUE(MessageDifferencer::Equals(expected_payload, payload));
 }
 
-TEST(AuthnUtilsTest, ProcessJwtPayloadWithNoAudTest) {
+TEST(AuthnUtilsTest, ProcessJwtPayloadWithAudListTest) {
```

Comment:
Why ProcessJwtPayloadWithNoAudTest() is replaced by ProcessJwtPayloadWithAudListTest(), instead of having both of these two tests?

Reply:
The ProcessJwtPayloadWithNoAudTest is testing when the aud field is not in the token which is not very useful, we don't need to test a field is not in the token because we always compare it with an expected output. If there anything not in the expected output, the test will fail, so we don't need to specifically have a `NoAudTest`.